src/hooks/index.ts

```diff
@@ -23,6 +23,8 @@ import {processTxHistoryData} from '../legacy/processTransactions'
 import {WalletMeta} from '../legacy/state'
 import storage from '../legacy/storage'
 import {cardanoValueFromRemoteFormat} from '../legacy/utils'
+import {useSearch} from '../Search'
+import {useSelectedWallet} from '../SelectedWallet'
```

Comment:
`hooks.index.ts` will be extracted into another library so it can be shared with extension
so, no additions that are tied to mobile

Reply:
➕